tests/analysis/indirect/test_losses.py

```diff
@@ -43,58 +38,67 @@ def test_initialize(self):
         assert isinstance(_losses, Losses)
         assert isinstance(_losses.analysis, AnalysisSectionIndirect)
 
-    def test_traffic_shockwave(self):
+    def test_initialize_with_data(self):
         # 1. Define test data
         _config = AnalysisConfigWrapper()
         _config.config_data.input_path = Path("sth")
-        _analysis = AnalysisSectionIndirect(
-            duration_event=60,
-            duration_disruption=None,
-            fraction_detour=None,
-            fraction_drivethrough=24,
-            rest_capacity=42,
-            maximum_jam=None,
-            partofday=None,
+        _analyses = AnalysisSectionIndirect(
+            part_of_day=None,
+            resilience_curve_file=test_data
+            / "losses"
+            / "csv_data_for_losses"
+            / "resilience_curve.csv",
+            traffic_intensities_file=test_data
+            / "losses"
+            / "csv_data_for_losses"
+            / "traffic_intensities.csv",
+            values_of_time_file=test_data
+            / "losses"
+            / "csv_data_for_losses"
+            / "values_of_time.csv",
+            name="single_link_redundancy_losses_test",
```

Comment:
Make your life easier:

```suggestion
        _losses_csv_data = test_data.joinpath("losses", "losses", "csv_data_for_losses")
        _analyses = AnalysisSectionIndirect(
            part_of_day=None,
            resilience_curve_file=_losses_csv_data.joinpath("resilience_curve.csv"),
            traffic_intensities_file=_losses_csv_data.joinpath("traffic_intensities.csv"),
            values_of_time_file=_losses_csv_data.joinpath("values_of_time.csv"),
            name="single_link_redundancy_losses_test",
```